<launch>
    <node pkg="calib_image_saver" type="singleImageSaver" name="saver" output="screen">
        <remap from="/image_input" to="/djiros/image"/>
        <param name="image_path" type="string" value="/home/nvidia/images"/>
        <param name="image_name" type="string" value="IMG_"/>
        <param name="rate" type="int" value="9"/>
        <param name="board_width" type="int" value="9"/>
        <param name="board_height" type="int" value="8"/>
        <param name="is_use_OpenCV" type="bool" value="false"/>
        <param name="is_show" type="bool" value="true"/>
    </node>

</launch>

